tests/config.spec.coffee

```diff
@@ -4,6 +4,7 @@ timekeeper = require('timekeeper')
 resin = require('resin-sdk-preconfigured')
 errors = require('resin-errors')
 deviceConfig = require('../lib/config')
+_ = require('lodash')
```

Comment:
Nit: I think we only support Node 4+ for everything now, so you can just use `Object.assign`, we don't _need_ lodash here.